### PR TITLE
Collage refactor

### PR DESCRIPTION
**Pull request by AndreasMadsen**
* Refactors collage to scale the logo more dynamically. The previous version had some heavy jitter when the resolution was changed.
* Most importantly, downsize all profile pictures dramatically. The resolution is still 2x the size and therefore renders beautifully on high-resolution displays (phones etc). The size and compression before were just much higher than what could be displayed. 
* Use lazy-loading for all profile pictures.
* Use `.webp` format, with `.jpg` as a fallback, for improved compression.

This should dramatically reduce the loading time of the font page and member page, which previously could be several seconds. -- The current download size is 7.4MB, this reduces it to 187kb.

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -211,7 +211,7 @@
 
 - name: Shabnam Haghzare
   title: intern
-  picture: Shabnam_Haghzare.JPG
+  picture: shabnam_haghzare
```
Comment (dalafh) [inline _data/people.yaml:214]: Is there a reason why Shabnam is the only one with an _ in her name?

File: _includes/collage.html [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+        <img class="collage-title" src="/assets/images/logo_{{site.lang}}.svg">
 
         <div class="team-members">
             {% for person in site.data.people %}
             <div class="member-details"
-                {% if person.collage_column_start != nil %}
-                style="grid-column-start: {{person.collage_column_start}};"
-                {% endif %}
-            ><img src="/assets/images/bios/{{person.picture}}"></div>
+            {% if person.collage_column_start != nil %}
```
Comment (dalafh) [inline _includes/collage.html:8]: Is it possible to fix the alignment from line 8 to 10 since it's in the div tag?

File: _includes/collage.html [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+            style="grid-column-start: {{person.collage_column_start}};"
+            {% endif %}
+            >
+            <picture>
```
Comment (dalafh) [inline _includes/collage.html:12]: Same for lines 12 to 16 that are between the two div tags

File: _includes/collage.html [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
+                <source srcset="/assets/images/bios_compressed/{{person.picture}}.webp" type="image/webp">
+                <source srcset="/assets/images/bios_compressed/{{person.picture}}.jpg" type="image/jpeg">
```
Comment (dalafh) [inline _includes/collage.html:14]: Are the lines 14 and 15 redundant since they both use exactly the same source?

File: _includes/person.html [hunk 5]
```diff
@@ -1,5 +1,9 @@
 {% if include.picture != "" and include.picture != nil %}
-    <img src="/assets/images/bios/{{include.picture}}">
+<picture>
```
Comment (dalafh) [inline _includes/person.html:2]: Fixing alignment inside the if (lines 2 to 6)

File: _includes/person.html [hunk 6]
```diff
@@ -1,5 +1,9 @@
...
+    <source srcset="/assets/images/bios_compressed/{{include.picture}}.webp" type="image/webp">
+    <source srcset="/assets/images/bios_compressed/{{include.picture}}.jpg" type="image/jpeg">
```
Comment (dalafh) [inline _includes/person.html:4]: Same as before: are the lines 4 and 5 redundant since they both use exactly the same source?

File: assets/css/style.scss [hunk 7]
```diff
@@ -66,7 +69,7 @@ ul.people li p {
 	margin-bottom: 20px;
 
 	display: grid;
-	grid-template-columns: auto auto;
```
Comment (dalafh) [inline assets/css/style.scss:69]: Why is that necessary?

File: assets/css/style.scss [hunk 8]
```diff
@@ -97,48 +100,12 @@ ul.people li p {
...
-		font-size: 80px;
-	}
-	.collage-title-text {
-		font-size: 70px;
-	}
-}
 
 @media only screen and (max-width : 570px) {
 	.collage {
-		margin-top: 0px;
```
Comment (dalafh) [inline assets/css/style.scss:140]: Why is that necessary?

**Review by dalafh — COMMENTED** [review]
Why are there white.jpg and white.webp in the assets/images/bios_compressed folder?

File: _data/people.yaml (hunk 1 above)
Comment (AndreasMadsen) [reply _data/people.yaml:214]: I don't think so. The file was named this way before I took over.

Comment (dalafh) [reply _data/people.yaml:214]: Ohhh okay yeah

File: _includes/person.html (hunk 6 above)
Comment (AndreasMadsen) [reply _includes/person.html:4]: No. The `src` in `img` is a fallback, in case the browser doesn't support `picture`.

File: assets/css/style.scss (hunk 7 above)
Comment (AndreasMadsen) [reply assets/css/style.scss:69]: Because the two child nodes are now both flexible, hence auto will have arbitrary meaning. Before the size of the `title` child contained this enough to have a deterministic layout.

Comment (dalafh) [reply assets/css/style.scss:69]: I see!

File: _includes/person.html (hunk 5 above)
Comment (AndreasMadsen) [reply _includes/person.html:2]: Indentation of template files is never perfect. I general, the indentation should be done such the produced content is indented correctly.

Comment (dalafh) [reply _includes/person.html:2]: Yeah okay that makes sense. I know it doesn't change the logic but Sarath said that the reviews needed to focus more on the coding practices than the logic so that's why I pointed it out. I agree that it's not necessary though

File: assets/css/style.scss (hunk 8 above)
Comment (AndreasMadsen) [reply assets/css/style.scss:140]: Fixed.

File: _includes/collage.html (hunk 2 above)
Comment (AndreasMadsen) [reply _includes/collage.html:8]: Fixed.

File: _includes/collage.html (hunk 3 above)
Comment (AndreasMadsen) [reply _includes/collage.html:12]: Fixed.

**Comment by AndreasMadsen** [comment]
> Why are there white.jpg and white.webp in the assets/images/bios_compressed folder?

Previously the layout was dependent on everybody having a profile image. If someone didn't want that, they could use `white.png`. I've fixed it now such it is completely redundant and removed the files.

File: assets/css/style.scss [hunk 9]
```diff
@@ -97,48 +100,13 @@ ul.people li p {
...
-	}
-	.collage-title-text {
-		font-size: 70px;
-	}
-}
 
 @media only screen and (max-width : 570px) {
 	.collage {
 		margin-top: 0px;
-		margin-bottom: 20px;
```
Comment (dalafh) [inline assets/css/style.scss:141]: Same for that one, sorry for noticing a little late...

Comment (AndreasMadsen) [reply assets/css/style.scss:141]: This line didn't do anything, because `margin-bottom: 20px` always applied.

Comment (dalafh) [reply assets/css/style.scss:141]: Okay great then!

**Review by dalafh — APPROVED** [review]
All good!
